contentcuration/contentcuration/assets/channelList/mixins.js

```diff
@@ -1,7 +1,6 @@
-import _ from 'underscore';
+import omit from 'lodash/omit';
```

Comment:
Are modular imports the big reason for switching to lodash?

Reply:
Yes - underscore still doesn't support them.